src/Swarm/Language/Capability.hs

```diff
@@ -253,39 +169,17 @@ constCaps = \case
   Exp -> Just CArith
   Whoami -> Just CWhoami
   Self -> Just CWhoami
-  Swap -> Just CSwap
-  Atomic -> Just CAtomic
-  Instant -> Just CGod
```

Comment:
I think this change will significantly simplify keeping `execCaps` up to date. 🙂 

As it is now clear to see we had `Instant` and `Use` in the wrong sections.